15-draft.json

```diff
@@ -417,6 +417,53 @@
             ]
           }
         },
+        "carbondioxide": {
+          "description": "CO2 sensor",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "value": {
+                "description": "The sensor value",
+                "type": "number"
+              },
+              "unit": {
+                "description": "The unit of the sensor value.",
+                "type": "string",
+                "enum": [
+                  "ppm"
```

Comment:
This is the most common unit. I wonder if we should add `vol%` as the [STC31-C](https://sensirion.com/products/catalog/STC31-C) uses it (because it measures it via the thermal conductivity).